### PR TITLE
Update to egui 0.25 and winit 0.29

### PR DESCRIPTION
**Pull request by emilk**
### What
@Wumpf please test the `re_renderer` examples.

There's a bunch of new duplicated dependencies, which sucks. I've tried to ping the maintainers of the offending packages.

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [app.rerun.io](https://app.rerun.io/pr/4732/index.html)
  * Using examples from latest `main` build: [app.rerun.io](https://app.rerun.io/pr/4732/index.html?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [app.rerun.io](https://app.rerun.io/pr/4732/index.html?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG

- [PR Build Summary](https://build.rerun.io/pr/4732)
- [Docs preview](https://rerun.io/preview/413e96bf89e8e2c4b5b1d27f67c08cfeff924ba5/docs) 
- [Examples preview](https://rerun.io/preview/413e96bf89e8e2c4b5b1d27f67c08cfeff924ba5/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -230,7 +230,7 @@ wasm-bindgen-futures = "0.4.33"
 web-sys = "0.3.61"
 web-time = "0.2.0"
 webbrowser = "0.8"
-winit = "0.28.7"
+winit = { version = "0.29.9", efault-features = false }
```
Comment (Wumpf) [inline Cargo.toml:233]: I know you like to prefix your own libraries with `e`, but this goes too far! 😄

**Comment by Wumpf** [comment]
> @Wumpf please test the re_renderer examples.

examples no longer animate/request new frames. E.g. check `cargo run -p re_renderer_examples --bin 2d`, the right side is supposed to rotate but it only refreshes when resizing

**Comment by emilk** [comment]
The wasm build of `re_renderer_examples` is also broken. I'm investigatng.

**Review by Wumpf — APPROVED** [review]
lgtm!
let's briefly hold on with this until the release is done :)

**Pull request merged**